tests/acceptance/features/apiSpaces/createSpace.feature

```diff
@@ -0,0 +1,154 @@
+@api
+Feature: create space
+  Only user with admin and SpaceAdmin permissions can create new spaces
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario Outline: The user without permissions to create space cannot create a Space via Graph API
```

Comment:
@amrita-shrestha there are a lot of inconsistencies in this PR in scenario titles like
```
The user without permissions to create space cannot create a Space via Graph API
user can disable their own Space via the Graph API
User with role User and Guest cannot disable others Space via the Graph API
```
can we follow a similar pattern, if this exists in other existing scenarios as well maybe we might need to refactor them to follow a similar pattern.

I think we normally use `user ....` without capitalization and without an article.

This is not a big problem but would be nice to be consistent. 
Others can also say what we would like moving forward 

@PrajwolAmatya @grgprarup @SagarGi @saw-jan

Reply:
I see that you've just copy pasted the existing scenarios. So, maybe we need to discuss if we want the similar pattern or it doesn't matter

Reply:
Yeah, better to be consistent

Reply:
I think we can do that in a separate issue. This way it will be sure that all scenario titles follow the same pattern otherwise we gonna forget it. After some time it was all same again.  @saw-jan @SwikritiT